apps/app/pages/admin/lectures/index.tsx

```diff
@@ -0,0 +1,164 @@
+  const collapsed = true;
+
+  useEffect(() => {
+    api
+      .listEvents()
+      .then((response: any) => setEvents(response.data))
+      .catch(() => {});
+  }, []);
+
+  useEffect(() => {
+    api
+      .listLectures()
+      .then((response: any) => setLectures(response.data))
+      .catch(() => {});
+  }, []);
+
+  useEffect(() => {
+    if (selectedEvent != "") {
+      setSelectedLectures(
+        lectures.filter((lecture) => lecture.event.id === selectedEvent)
+      );
+    }
+  }, [selectedEvent, lectures]);
+
+  return (
+    <AppLayout>
+      <Row justify="space-between">
+        <Title level={2}>Criar uma Lecture</Title>
+        <Link href="/admin/lectures/new">
+          <Button
+            shape="circle"
+            type="primary"
+            size="large"
+            icon={<PlusOutlined />}
+          />
+        </Link>
+      </Row>
+      <Row justify="space-around" gutter={[10, 10]}>
+        <Select
+          defaultValue="Selecione um Evento"
+          style={{ width: 200 }}
+          onChange={setSelectedEvent}
+          value={selectedEvent == "" ? undefined : selectedEvent}
+        >
+          {events.map((event) => (
+            <Select.Option value={event.id}>{event.title}</Select.Option>
+          ))}
+        </Select>
+      </Row>
+      {selectedLectures.map((lecture) => (
+        <Card
+          title={lecture.event?.title}
+          style={{ maxWidth: 460, margin: 15 }}
+        >
+          <Space
+            align="end"
+            direction="horizontal"
+            wrap={(screens.sm && !screens.md) || screens.xs}
+          >
+            <Descriptions size="small" column={1} layout="horizontal">
+              <Descriptions.Item
+                labelStyle={labelStyle}
+                label={
+                  <span>
+                    <EnvironmentOutlined /> Ninja
+                  </span>
+                }
+                span={1}
+              >
+                <Link href={`/profile/ninja/${lecture.ninja?.id}`}>
+                  {lecture.ninja?.first_name + " " + lecture.ninja?.last_name}
+                </Link>
+              </Descriptions.Item>
+              <Descriptions.Item
+                labelStyle={labelStyle}
+                label={
+                  <span>
+                    <HomeOutlined /> Mentor
+                  </span>
+                }
+              >
+                <Link href={`/profile/mentor/${lecture.mentor?.id}`}>
+                  {lecture.mentor?.first_name + " " + lecture.mentor?.last_name}
+                </Link>
+              </Descriptions.Item>
+              <Descriptions.Item
+                labelStyle={labelStyle}
+                label={
+                  <span>
+                    <EnvironmentOutlined /> Localização
+                  </span>
+                }
+                span={1}
+              >
+                Departamento de Informática, Edifício 7, Universidade do Minho
```

Comment:
Here should be the current event location (`lecture.event.location`). Not an hard coded one.